src/screens/ImportSeedPhraseSheet.tsx

```diff
@@ -56,83 +53,52 @@ export const ImportSeedPhraseSheet: React.FC = () => {
   });
   const labelTertiary = useForegroundColor('labelTertiary');
 
-  const [keyboardVisible, setKeyboardVisibility] = useState(true);
-
-  useEffect(() => {
```

Comment:
seems theres you removed a few instances of custom keyboard logic which I love but wanna confirm why